.github/workflows/installcheck.yaml

```diff
@@ -2,9 +2,9 @@ name: Build / Regression
 
 on:
   push:
-    branches: [ 'master', 'PG15' ]
+    branches: [ 'master', 'pg16' ]
```

Comment:
All `pg16 ` entries need to be uppercase `PG16`. Except where that `pg15` is lowercase.